FAST by TRAVIS TIGHE

tt1026 opened the pull request:
So far, I’m struggling with getting movements to repeat once they’re
done. In the future, there will be far more elements of the environment
rushing past the user’s pov, and all rushing away from a vanishing
point, not down the x axis. The pov will also shake the faster you go, to simulate turbulence.

tt1026 committed:
FAST by TRAVIS TIGHE

So far, I’m struggling with getting movements to repeat once they’re
done. In the future, there will be far more elements of the environment
rushing past the user’s pov, and all rushing away from a vanishing
point, not down the x axis.

LadyK commented:
You have: if (int x = 800){} // which means that we have "x", which is integer size, and holds 800.
Remember if statements test for truth - boolean logic. What ever is between the () is going to be true or false. We don't typically set variable or values there.

Try: if (x == 800){}

In the mean time, keep programming. I'm looking forward to seeing your progress next week.

tt1026 commented:
Hey thanks! Didn't think about that distinction. I'm still getting an error
on the if statement, it's giving me "error in 'int'". Not really sure what
to make of it, and all my syntax seems correct.

Best,
TT

On Tuesday, September 22, 2015, LadyK notifications@github.com wrote:

> You have: if (int x = 800){} // which means that we have "x", which is
> integer size, and holds 800.
> Remember if statements test for truth - boolean logic. What ever is
> between the () is going to be true or false. We don't typically set
> variable or values there.
> 
> Try: if (x == 800){}
> 
> In the mean time, keep programming. I'm looking forward to seeing your
> progress next week.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IDMUY1133BCreativeCodingAssignments/Project_1_Adjective/pull/10#issuecomment-142471504
> .

LadyK commented:
Unless I'm miss understanding you, take a look again:

Try: if (x == 800){}

Take out the "int" and add a "=="

Otherwise, push up new code so I can see where your issue is.

tt1026 committed:
9/29 update

Changing the concept a bit, going to animate a spaceship cockpit pov,
which will shake with turbulence the faster you go

LadyK commented:
Hi Travis,

Check out the Nature of Code text. It's online and there is a reference to it in the resources page. I think many of the concepts in there would definitely enhance your work. Think about force, speed, and time with reference to motion. Think about different states that users could control, or that just happen over time. How could this experience vary? How many different forms of fast can you reflect?

I'd also like to see custom functions that pass parameters and arrays. I think those can be quite easily incorporated.
